stestr/cli.py

```diff
@@ -68,14 +68,14 @@ def _set_common_opts(self, parser):
                                  "command. If one isn't specified then "
                                  ".stestr.conf in the directory that a command"
                                  " is running from is used")
-        parser.add_argument('--repo-type', '-r', dest='repo_type',
-                            choices=['file', 'sql'], default='file',
-                            help="DEPRECATED: Select the repo backend to use")
         parser.add_argument('--repo-url', '-u', dest='repo_url',
                             default=None,
                             help="Set the repo url to use. An acceptable value"
```

Comment:
I feel it's good to remove this sentence to avoid confusion - "An acceptable value for this depends on the repository type used.". New users don't know about the ``--repo-type`` option any more.

Reply:
Fixed in: https://github.com/mtreinish/stestr/pull/318/commits/46e4c46be25c0bbc6668ced292ac45c09fef7b45